src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+import base.Figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double side;
+    private final double firstBase;
+    private final double secondBase;
+
+    public IsoscelesTrapezoid(String color, double side, double firstBase, double secondBase) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return firstBase * side - secondBase * side;
```

Comment:
The formula for the area of an isosceles trapezoid is incorrect. It should be the average of the two bases times the height, not the product of the side and the difference of the bases. You need to calculate the height first and then apply the correct formula.